source/Octopus.Server.Client/Model/SelectedGitResource.cs

```diff
@@ -25,4 +25,10 @@ public SelectedGitResource(string actionName, GitReferenceResource gitReferenceR
     /// The selected Git reference
     /// </summary>
     public GitReferenceResource GitReferenceResource { get; set; }
+    
+    /// <summary>
+    /// The name of the Git resource reference
+    /// </summary>
+    /// <remarks>May be empty for steps which have a primary Git resource</remarks>
+    public string GitResourceReferenceName { get; set; } = "";
```

Comment:
What does adding this allow us to do?

Reply:
This makes the difference between primary and secondary (etc) git resources.   Providing this name makes it non-primary, leaving it blank makes it a primary.

See the equivalent property for Packages here: 
https://github.com/OctopusDeploy/OctopusClients/blob/f6d191220b086ebdc4c2684ed8e1296c790fffca/source/Octopus.Server.Client/Model/SelectedPackage.cs#L45